cli/Sources/ProjectDescription/CacheProfiles.swift

```diff
@@ -0,0 +1,58 @@
+extension Tuist {
+    /// Cache profile type.
+    public enum CacheProfileType: Codable, Equatable, Sendable {
```

Comment:
I'm inclined to hoist these at the root (i.e. not inside a `Tuist` extension). It's something that we've done in other places but:
- There's no need for symbol-scoping to avoid potential conflicts
- We are unnecessarily making things hard for the compiler.